torch/distributed/distributed_c10d.py

```diff
@@ -378,7 +378,8 @@ def init_process_group(backend,
     if backend == Backend.MPI:
         if world_size != -1 or rank != -1:
             warnings.warn(
-                "For MPI backend, world_size and rank are ignored "
+                f"For MPI backend, world_size ({world_size}) "
```

Comment:
Lint failure is real. We are deprecating PY2 soon, but let's follow linter's suggestion when it is still in place, i.e., replace `f""` with `"".format()`

Reply:
Thank you! Let me update and refresh the PR.